tests/math/math.cpp

```diff
@@ -20,6 +20,20 @@ limitations under the License.
 
 #include <libcellml>
 
+// Depending on the libXML2 version the error messages differ.
+const std::vector<std::string> expectedIssues_2_9_4 = {
```

Comment:
Is 2.9.4 (and 2.9.10 below) related to the version of libxml2 used? If so, it might be worth making it clear or we might, in the future, wonder what it is about?

Reply:
I had completely overlooked the original message that mentioned `libxml2` (and not `libXML2` or even `LibXml2`). Still, I was replacing `expectedIssues_2_9_4` with something like `expectedIssues_libxml2_2_9_4`. Anyway, I guess the comment is sufficient. So, resolving this issue.